src/renderers/react/WrapperControls.tsx

```diff
@@ -285,9 +305,24 @@ export const RightActionControls = ({
           <SvgChevronRightDouble />
         </div>
       </Button>
-    </RightActions>
-  );
-};
+      <Button
+        type="button"
+        data-cy={moveDownTestId}
+        expanded
+        disabled={!moveDown(false)}
+        onClick={() => {
+          sendTelemetryEvent?.(CommandTelemetryType.PMEDownButtonPressed, {
+            jump: false,
+          });
+          moveDown(true);
+        }}
+        aria-label="Move element down"
+      >
+        <SvgArrowDownStraight />
+      </Button>
```

Comment:
This button is re-ordered to account for some column / column-reverse shenanigans.